### DISCUSSION TITLE
Complicated Formula requires more than "Count" Function

### DISCUSSION THREAD

**Discussion by JHRI2** [remarkablemark/jira-dashboard-gadget-issue-formula#500] [Q&A]
I am trying to create a Gadget to express a "Complicated" formula.

Specifically, for a list of Issues matching this JQL: project = "SPWHADC" AND issuetype != Epic OR labels = SPWHADC

I need to then look for the SUM of the WorkRatio for that list...

Then AVERAGE/100. 

Is that possible with this Gadget?

**Comment by remarkablemark** [comment]
@JHRI2 This gadget may not achieve your use-case since it counts the sum of Jira tickets and isn't able to get values from the fields.

**Reply by JHRI2** [reply]
Fair enough; thank you!
